### DISCUSSION TITLE
Welcome to Pikka Discussions!

### DISCUSSION THREAD

**Discussion by rsayfutdinov** [rsayfutdinov/Pikka#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to improve Pikka. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Suggest improvements
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

Don't hesitate to open new discussions.

**Comment by ycikdevelopment** [comment]
Hi, not sure if this is the correct space to comment.

I bought this app awhile ago. Updated to the latest version a few days ago to my Mac Book. Cannot use it wants me to purchase it again?

I clicked on "Restore Purchases" but the same result.
![Screenshot 2021-06-17 at 13 56 26](https://user-images.githubusercontent.com/48251222/122392193-f398d400-cf73-11eb-8f92-e05965ad2798.png)
![Screenshot 2021-06-17 at 13 56 17](https://user-images.githubusercontent.com/48251222/122392210-f72c5b00-cf73-11eb-8ba4-2249ef123910.png)

**Reply by rsayfutdinov** [reply]
Hello,

Sorry that it didn't work for you immediately.

There is two options:

1) I'm already working on an update, that hopefully fix your issues. It will be available on the app store within next 4-7 days. So you need to wait a little.

2) I drop the price to 0.99 for a while, you purchase it again. And I refund the purchase via Pay Pal. I know it’s not perfect solution but, should work very quick. Let me know if you want to go with this one.

Thanks for understanding ,
Ruslan.

**Reply by ycikdevelopment** [reply]
Hi there, so I have the updated version of the color picker but still it displays the prompt as seen attached.

How do I get this to work on my MacBook. It did work well in the past.

<img width="551" alt="Screenshot 2021-06-25 at 10 06 01" src="https://user-images.githubusercontent.com/48251222/123392134-02e3d700-d59d-11eb-9467-b5ea743a9b51.png">
![Screenshot 2021-06-25 at 10 06 09](https://user-images.githubusercontent.com/48251222/123392144-05463100-d59d-11eb-8deb-9a9d1ebafd37.png)

**Reply by rsayfutdinov** [reply]
> Hi there, so I have the updated version of the color picker but still it displays the prompt as seen attached.
> 
> How do I get this to work on my MacBook. It did work well in the past.
> 
> <img alt="Screenshot 2021-06-25 at 10 06 01" width="551" src="https://user-images.githubusercontent.com/48251222/123392134-02e3d700-d59d-11eb-9467-b5ea743a9b51.png">
> 
> ![Screenshot 2021-06-25 at 10 06 09](https://user-images.githubusercontent.com/48251222/123392144-05463100-d59d-11eb-8deb-9a9d1ebafd37.png)

Hi,
Please reach me at support@1sense.app. I will send you a promo code, so you don't have to pay for the app again.

Regards,
Ruslan

**Comment by ycikdevelopment** [comment]
Cool, your response is much appreciated. I will wait for the update.
 

Regards | Groete

Regardt Ferreira
Managing

	M: +27(0)82 7733 499
E: ***@***.***
A: George, South Africa
E: ***@***.***
W: www.ycik.co.za <https://www.ycik.co.za/>
Website Design | Hosting & Registrations | Website Optimization (SEO)| Content Optimization (SEO)| Search Engine Marketing | Social Media Marketing | Mobile Marketing and Integration | Placement of Online Ads | Email Marketing | Online Branding | Copy writing | Funnel Marketing

CONFIDENTIALITY/DISCLAIMER: The contents of this e-mail are confidential and are not to be regarded as a contractual offer or acceptance from YCIK Development cc, (registered in South Africa). If you are not the addressee, or if this has been copied or sent to you in error, you must not use data herein for any purpose, you must delete it, and should inform the sender. YCIK Development disclaims liability for accuracy or completeness, and opinions expressed are those of the author alone. YCIK Development may monitor communications. Third party rights acknowledged. © 2006-2015.Emails cannot be guaranteed to be secure or free of errors or viruses. The sender does not accept any liability or responsibility for any interception, corruption, destruction, loss, late arrival or incompleteness of or tampering or interference with any of the information contained in this email or for its incorrect delivery or non-delivery for whatsoever reason or for its effect on any electronic device of the recipient. If verification of this email or any attachment is required, please request a hard-copy version.

**Reply by rsayfutdinov** [reply]
Hey,
Reach me at support@1sense.app
I will send you a promo code to use in the app. So you do not need to wait for an update.
Ruslan.

**Comment by SNicker317** [comment]
Just Purchased the Pro version of 2.2.1 and there is one issue I've found and a possible suggestion below. btw, I see no real immediate difference from the free non-Pro version, with the exception that the upgrade to Pro option has been removed from the window, just started using so, I could be wrong.

1. Changing the " Show picked color information for " 3 secs... Constantly reverts back to 3 secs and any new values entered have no effect on the display time.
2. Picking a color on the screen, would be great if this also opened a popup such as when Creating a color, that could allow for the display of Complementary colors, Analguous etc... to that color

 Regards,
      Steve

**Reply by rsayfutdinov** [reply]
Hey Steve, 

1. Thanks for the bug report. It is already fixed. The update will be available on the Appstore soon.
2. About feature request. Do you mean to show something like this after picking a color? 
<img width="362" alt="Screenshot 2021-06-18 at 10 04 35" src="https://user-images.githubusercontent.com/12187524/122528499-afadd980-d01c-11eb-890a-2a068be8d844.png">

I think that will bother the users, who just want to quickly copy/paste some color values from the screen. 

I could try to show the popup while holding cmd or option key while picking a color.
I would definitely think about it.

Ruslan

**Comment by ZeoSonE** [comment]
Hi there!
Apologies in advance in case I have posted all this into the wrong thread.
My first issue is regarding the Pro version upgrade.
I really don't mind to support developers in many cases, but just need to understand it.
Is that upgrade for the existing customers or new ones.Since I have purchase full app, it sound a bit harsh.
I do understand that this a new layer of functionalities, but I had previously a libraries enabled in, this is been disabled and this sounds like removing functionality from the paid app which is definitely not right.

<img width="386" alt="Screenshot 2021-06-25 at 12 03 44" src="https://user-images.githubusercontent.com/7634694/123415697-8c9ba080-d5ad-11eb-91ca-ba6458f10d8a.png">

If I'll click to add another library here
<img width="95" alt="Screenshot 2021-06-25 at 12 03 01" src="https://user-images.githubusercontent.com/7634694/123415706-90c7be00-d5ad-11eb-86df-76d3355002b9.png">

It displaying the popup window with in-app purchase
<img width="550" alt="Screenshot 2021-06-25 at 12 05 52" src="https://user-images.githubusercontent.com/7634694/123415837-b785f480-d5ad-11eb-9639-94269ea0cd49.png">

Again, agree to support, but without removing functionality and in mind of the existing customer like and upgrade price.

Now Regarding the functionality and UX

1. If I capture new colour, it goes as a last in the list, most right one in the library.
I think it's more logical to add it vie versa on the most left please, as we are counting from left to right and left taking always a first place.
Again, this could be an option in a setting "order of the captured colours" Left > Right, and Right > Left

2. You could also make an option disappearing off the panel when clicked outside of it. I prefer the panel gone after doing inside it's area.

Let me know if any comments were useful.

Thanks

**Reply by ZeoSonE** [reply]
@rsayfutdinov Bothering you again. I was checked my purchases history and found your app.
I have downloaded it for free on May 2017, so please skip my first point in the previous post. Apologies for that.

**Reply by rsayfutdinov** [reply]
Hey,

It was not my intention to limit the functionality for existing customers. It seems to be a bug in the app, that I cannot fix at the moment, unfortunately. So i give the promo codes for the users, who already purchased the app and have this issue.

> You could also make an option disappearing off the panel when clicked outside of it. I prefer the panel gone after doing inside it's area.

> If I capture new colour, it goes as a last in the list, most right one in the library.
I think it's more logical to add it vie versa on the most left please, as we are counting from left to right and left taking always a first place.
Again, this could be an option in a setting "order of the captured colours" Left > Right, and Right > Left

Okay. Seems to be logical. Took this to my backlog.

Thanks for the feedback,
Ruslan.

**Reply by ZeoSonE** [reply]
Hi Russian,

Thanks for the quick reply.
That would be nice to have a discount code for the upgrade purchase.

Thanks!

**Reply by rsayfutdinov** [reply]
Please contact me at support@1sense.app
